### PR TITLE
Metrics server

### PR DESCRIPTION
**Pull request by ktsakalozos**
This is the metrics server addon.

Fixes https://github.com/ubuntu/microk8s/issues/53

### CODE REVIEW

File: tests/test-live-addons.py [hunk 1]
```diff
@@ -67,3 +68,9 @@ def test_forward(self):
         """
         validate_forward()
 
+    def test_metrics_server(self):
+        """
+        Validates port forward works.
```
Comment (tvansteenburgh) [inline tests/test-live-addons.py]: s/port forward/metrics server/

File: tests/test-upgrade.py [hunk 2]
```diff
@@ -91,6 +92,14 @@ def test_upgrade(self):
         except:
             print('Will not test port forward')
 
+        try:
+            enable = microk8s_enable("metrics-server")
+            assert "Nothing to do for" not in enable
+            validate_metrics_server()
+            test_matrix['metrics_server'] = validate_metrics_server
+        except:
+            print('Will not the metrics server')
```
Comment (tvansteenburgh) [inline tests/test-upgrade.py]: s/the/test/

**Comment by ktsakalozos** [comment]
The metrics server is not enabled by default. It is enabled as a normal addon so I added it in the list of the addons in the README (which I should have done in the first place but forgot).

**Comment by ktsakalozos** [comment]
Thank you for the review.

**Pull request merged**